arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -854,8 +854,8 @@
           "type": "string",
           "description": "Permissions for the key -- read-only or full permissions.",
           "enum": [
-            "READ",
-            "FULL"
+            "Read",
+            "Full"
```

Comment:
@veronicagg, @johanste - With these changes the data validation error for enum mismatch will go away. @jhendrixMSFT, @markcowl, @lmazuel - Heads up. This is a good breaking change. It change the valid enum values in the new version of storage management sdk in all the languages. Since all the properties are readOnly, the effect would be minimal. I assume there shouldnt be any breaking change in Powershell and CLI (api wise).